docs/source/configuration/environmentvariables.md

```diff
@@ -186,6 +186,10 @@ You can also generate builds on your continuous integration, then deploy them an
     ```
 
 `SITE_DEFAULT_LANGUAGE`
+    ```{deprecated} Volto 19
+    This environment variable was removed in Volto 19. Instead, Volto uses the default language configured in the backend.
```

Comment:
````suggestion
    This environment variable was removed in Volto 19.
    Instead, Volto uses the default language configured in the backend.

    ```{seealso}
    See {ref}`multilingual configuration in Volto <multilingual-volto-configuration-label>`.
    ```    
````